tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "commonjs",
+    "allowJs": true,
+    "checkJs": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "outDir": "types",
+    // "noImplicitAny": true,
```

Comment:
I opted this out to omit most of the work, which in turn means there are `any`s everywhere.